prow/config.yaml

```diff
@@ -116,6 +116,56 @@ presubmits:
       - name: cache-ssd
         hostPath:
           path: /mnt/disks/ssd0
+    run_after_success:
+    - name: pilot-e2e-smoketest
+      context: prow/pilot-e2e-smoketest.sh
+      always_run: true
+      rerun_command: "@istio-testing smoke test this"
+      trigger: "((?m)^@istio-testing (smoke )?test this,?(\\s+|$)|(?m)^/test( all| bazel),?(\\s+|$))"
+      skip_report: true
+      branches:
+      - master
+      spec:
+        containers:
+        - image: gcr.io/istio-testing/prowbazel:0.1.5
+          args:
+          - "--pull=$(PULL_REFS)"
```

Comment:
I think you want to do "--repo=istio/test-infra=master" instead of  "--pull=$(PULL_REFS)" since you are checking out istio master to run the test from.